docs/reference/src/main/asciidoc/ri-spi.asciidoc

```diff
@@ -979,3 +979,65 @@ Instead, this dependency needs to be deployed separately to the OSGi container.
 ** Deprecated `org.jboss.weld.environment.se.ShutdownManager` was removed.
 
 ** Deprecated `org.jboss.weld.environment.se.contexts.interceptors.ActivateThreadScope` was removed.
+
+==== Migration from Weld 2.x to 3.0
+
+Weld 3.0 implements CDI 2.0 so this is a huge update which contains bunch of new features.
+However, this document only covers part which might prove problematic while moving existing application from Weld 2.x to Weld 3.0.
+For more information on the CDI 2.0 itself, please consult http://docs.jboss.org/cdi/spec/2.0-PRD/cdi-spec.html[the specification]
+
+===== Java 8
+
+Both, Weld 3.0 and CDI 2.0 require Java 8 for compile time and runtime.
+
+===== Weld-core artifact change 
```

Comment:
This is a low priority change (most users don't even know this artifact exists ;-)... I would move it to the end of the list.